client/sites-dashboard/components/sites-dashboard.tsx

```diff
@@ -351,3 +352,25 @@ function useShowSiteCreationNotice( allSites: SiteExcerptData[], newSiteID: numb
 		window.history.replaceState( null, '', newUrl.toString() );
 	}, [ __, allSites, dispatch, newSiteID ] );
 }
+
+function useShowSiteTransferredNotice() {
+	const { __ } = useI18n();
+	const dispatch = useDispatch();
+	const shownSiteTransferredNotice = useRef( false );
+
+	useEffect( () => {
+		if ( shownSiteTransferredNotice.current ) {
+			return;
+		}
```

Comment:
@kozer , Do we need this ref `shownSiteTransferredNotice` ?
I tried removing this guard, and the variable on my local machine and it works nice. It displays the notice only once.